sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql

```diff
@@ -59,3 +59,12 @@ WHERE
     BETWEEN 84
     AND 87
   )
+  AND
+  -- See https://bugzilla.mozilla.org/show_bug.cgi?id=1803833
+  NOT (
+    event_category = 'nimbus'
+    AND event_method = 'validationFailed'
+    AND mozfun.map.get_key(event_map_values, 'reason') = 'invalid-feature'
+    AND mozfun.map.get_key(event_map_values, 'feature') IN ('nimbus-qa-1', 'nimbus-qa-2')
+    AND mozfun.norm.truncate_version(app_version, 'major') < 108
```

Comment:
Should this be `<= 108`? My understanding was that this would not be fixed before 109, but maybe I misinterpreted the discussion.

Reply:
The experiments are only running on 107.x and lower. But you are right, the bug causing this will still exist in 108